cucumber-tsflow/src/logger.ts

```diff
@@ -1,5 +1,8 @@
-import * as log4js from "log4js";
+import * as log4jsModule from "log4js";
+// @ts-expect-error
+import log4js from "log4js";
 
-const logger = log4js.getLogger("cucumber-js.tsflow");
+const getLogger = log4js?.getLogger || log4jsModule.getLogger;
```

Comment:
log4js has a similar issue when used by `"type": "module"` projects. This was my attempt to be backwards commutable for both versions of node's module loader.

Reply:
I believe `import {getLogger} from "log4js";` should work on both